gemrb/GUIScripts/pst/GUIINV.py

```diff
@@ -337,6 +339,10 @@ def UpdateSlot (pc, i):
 		Button.SetEvent (IE_GUI_BUTTON_ON_RIGHT_PRESS, InventoryCommon.OpenItemInfoWindow)
 		Button.SetEvent (IE_GUI_BUTTON_ON_SHIFT_PRESS, OpenItemAmountWindow)
 		Button.SetEvent (IE_GUI_BUTTON_ON_DRAG_DROP, OnDragItem)
+		#If the slot is being used to display the 'default' weapon, disable dragging.
+		if SlotType["ID"] == 10 and using_fists:
+			Button.SetEvent (IE_GUI_BUTTON_ON_PRESS, None)
```

Comment:
Shouldn't this be resetting IE_GUI_BUTTON_ON_DRAG_DROP?

Reply:
If you disable that event, you can't drag items onto the first quick weapon slot - this is just to stop you from picking up the hands/teeth like a regular item

Reply:
The stranger thing is that the fist items are not saved in the cre when I looked, they seem to be in fistweap.2da. So I don't know what part of the game is passing them to GemRB.DragItem...

Reply:
that's only for games with monks, but yeah, normal fists aren't added to the inventory.